### PR TITLE
stop using deprecated distro.linux_distribution

### PR DESCRIPTION
**Pull request by alexzorin**
`distro.linux_distribution` was deprecated (https://github.com/python-distro/distro/pull/296) in the release of `distro` at the end of last week. The deprecation is causing the `nopin` nightly tests to fail.

This change migrates Certbot off that function.

As far as I can tell, the Arch Linux edge case described in the code comments no longer happens, but better to be safe than sorry I think.

### CODE REVIEW

File: certbot/certbot/util.py [hunk 1]
```diff
@@ -390,13 +390,13 @@ def get_python_os_info(pretty=False):
     os_type, os_ver, _ = info
     os_type = os_type.lower()
     if os_type.startswith('linux') and _USE_DISTRO:
-        info = distro.linux_distribution(pretty)
+        distro_name, distro_version = distro.name() if pretty else distro.id(), distro.version()
         # On arch, distro.linux_distribution() is reportedly ('','',''),
```
Comment (bmw) [inline certbot/certbot/util.py:394]:
```suggestion
        # On arch, these values are reportedly the empty string so handle it
        # defensively
```

**Review by bmw — COMMENTED** [review]
Thanks a lot for jumping on this Alex.

Other than my inline suggestion to update the code comment, this LGTM.

**Review by bmw — APPROVED** [review]
